bin/git-effort

```diff
@@ -5,6 +5,7 @@ above=0
 # if the output won't be printed to tty, disable the color
 test -t 1 && to_tty=true
 color=
+columns=45
```

Comment:
`colums=45` seems duplicate now, because you will calculate it from longest filename later.